### PR TITLE
Feature team

### PR DESCRIPTION
**Pull request by austincarrjones**
## Type Of Change
- [X] New Feature
- [ ] Debugging
- [ ] Refactor/File Clean-up
- [X] Tests
- [ ] Documentation

## Provide A Summary Of Your Change
Created Teams and Teams_Factory classes and tests.

## Group Project Check List
- [X] I made test for my code, and made sure they were passing before submitting PR
- [X] I assigned at least 2 group members to review my PR

### CODE REVIEW

File: lib/team_factory.rb [hunk 1]
```diff
@@ -0,0 +1,19 @@
+class TeamFactory
+
+  def initialize
+    @teams = []
+  end
+  
+  def create_teams
```
Comment (noahdurbin) [inline lib/team_factory.rb:7]: Ijust realized that we don't have a fixture file for teams so this looks great.

File: spec/team_spec.rb [hunk 2]
```diff
@@ -0,0 +1,23 @@
+require 'spec_helper'
+
+RSpec.describe TeamFactory do
+
+  it 'exists' do
+    team = Team.new({team_id: 1, franchiseid: 23, team_name: "Atlanta United", abbreviation: "ATL", stadium: "Mercedes-Benz Stadium", link: "/api/v1/teams/1"})
+    expect(team).to be_a Team
+  end
+
+  it "has attributes from CSV header" do
```
Comment (noahdurbin) [inline spec/team_spec.rb:10]: looks good!

**Review by noahdurbin — APPROVED** [review]

File: lib/team_factory.rb [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+    CSV.foreach('./data/teams.csv', headers: true, header_converters: :symbol) do |row|
+      @teams << Team.new(
```
Comment (zach-bergman) [inline lib/team_factory.rb:9]: Nice, created these objects the same way Noah did. I did it differently, but it all works!

File: spec/team_spec.rb [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+    team_1 = Team.new({team_id: 1, franchise_id: 23, team_name: "Atlanta United", abbreviation: "ATL", stadium: "Mercedes-Benz Stadium", link: "/api/v1/teams/1"})
+    
+    # binding.pry
+    expect(team_1.team_id).to eq 1
+    expect(team_1.franchise_id).to eq 23
+    expect(team_1.team_name).to eq "Atlanta United"
+    expect(team_1.abbreviation).to eq "ATL"
+    expect(team_1.stadium).to eq "Mercedes-Benz Stadium"
+    expect(team_1.link).to eq "/api/v1/teams/1"
+  end
```
Comment (zach-bergman) [inline spec/team_spec.rb:21]: I like how you isolated one team to test the attributes! Very nice.

**Review by zach-bergman — APPROVED** [review]
All looks good!

**Pull request merged**